package.json

```diff
@@ -32,6 +33,7 @@
     "jquery": "2.1.4",
     "js-cookie": "2.0.4",
     "json-loader": "0.5.3",
+    "jsx-to-string": "^1.1.0",
```

Comment:
Should probably make this and `react-syntax-highlighter- below a `devDependency`, and make sure styleguide is only reachable in development.